src/AI/AIUtils.java

```diff
@@ -166,15 +167,15 @@ public static ArrayList<XYCoord> findEnemyUnits(Army army, GameMap gameMap)
    * Creates a map of COs to the units they control, based on what can be seen in the passed-in map.
    * Units owned by allies are ignored - ours can be trivially accessed via Commander.units, and we don't control ally behavior.
    */
-  public static Map<Commander, ArrayList<Unit> > getEnemyUnitsByCommander(Army myCommander, GameMap gameMap)
+  public static Map<Commander, ArrayList<Unit> > getEnemyUnitsByCommander(Army army, GameMap gameMap)
```

Comment:
Should this by `byArmy` now?

Reply:
Still no - individual units are assigned to a `Commander`, and their stats can differ within an `Army` as a result.